### ISSUE TITLE
json files ?

### ISSUE THREAD

**Issue by MohamedBsh** [danielbeach/sniffer#1]
Have you thought about doing this for JSON files? This repo is an excellent initiative to take advantage of the performance of Rust :))

**Comment by danielbeach** [comment]
@MohamedBsh good idea, yes I can add that support. Anything in particular you would want to see the `sniffer` tell you about json files?
